lib/project_copier.rb

```diff
@@ -61,7 +61,9 @@ def copy_project
     copied_project = project_to_copy.deep_clone include: INCLUDE_ASSOCIATIONS, except: EXCLUDE_ATTRIBUTES
     copied_project.owner = user
 
+    current_timestamp = Time.now.utc.strftime('%Y-%m-%d %H:%M:%S')
     copied_project.display_name += ' (copy)' if user == project_to_copy.owner
+    copied_project.display_name += " #{current_timestamp}"
```

Comment:
Following method used in workflow_copier.rb (see [workflow_copier.rb#L22](https://github.com/zooniverse/panoptes/blob/master/lib/workflow_copier.rb#L22)), and removing `if user == owner` (that doesn't seem necessary since _all_ workflows will have a unique `copy` tag added.

```suggestion
    copied_project.display_name += " (copy: #{Time.now.utc.strftime('%Y-%m-%d %H:%M:%S')})"
```